### PR TITLE
Update helloworld.c

### PR DESCRIPTION
**Pull request by Narutobapu**
Deliberately added error.

### CODE REVIEW

File: helloworld.c [hunk 1]
```diff
@@ -1,5 +1,6 @@
 #include<stdio.h>
 void main()
 {
-	printf("Hello World\n");
+	printf("Hello World");
```
Comment (ravikush11) [inline helloworld.c]: Add \n

File: helloworld.c [hunk 2]
```diff
@@ -1,5 +1,6 @@
...
+	return 0;
```
Comment (ravikush11) [inline helloworld.c]: remove return

**Review by ravikush11 — COMMENTED** [review]
Only 2 comments

**Comment by Narutobapu** [comment]
> Only 2 comments

OK.

File: helloworld.c (hunk 2 above)
Comment (Narutobapu) [reply helloworld.c]: Ok.

Comment (ravikush11) [reply helloworld.c]: cool

File: helloworld.c (hunk 1 above)
Comment (Narutobapu) [reply helloworld.c]: yeah, sure.

File: helloworld.c [hunk 3]
```diff
@@ -2,5 +2,4 @@
 void main()
 {
 	printf("Hello World");
```
Comment (ravikush11) [inline helloworld.c]: looks fine

**Review by ravikush11 — COMMENTED** [review]
no further comments please

File: helloworld.c (hunk 3 above)
Comment (Narutobapu) [reply helloworld.c]: Ok

**Review by ravikush11 — APPROVED** [review]

**Pull request merged**